Gemfile.on_prem.lock

```diff
@@ -218,6 +255,7 @@ DEPENDENCIES
   builder (= 3.2.3)
   codeclimate-test-reporter (~> 0.6.0)
   daemons (= 1.2.4)
+  falcon (~> 0.34)
```

Comment:
This pulls in loads of new gems related to async. For a future PR: look up [`install_if`](https://bundler.io/v2.0/man/gemfile.5.html#INSTALL_IF) in Gemfile to only pull these deps conditional on some manifest/configuration/envvar to enable async or sync (could also help ensure we don't depend on async code for sync, or on puma code when using falcon).

Reply:
:+1: